Data/Sector/Hong-Kong-Sector-File.ese

```diff
@@ -650,6 +659,14 @@ DISPLAY:VHHK·VHHK_SCT_07_DEH_03·135·250:VHHK·VHHK_SCT_07_DEH_03·135·250:VH
 DISPLAY:VHHK·VHHK_SCT_07_DEH_03·135·250:VHHK·VHHK_SCT_07_DEH_03·135·250:VHHK·VHHK_SCT_25_DEH_01·135·250
 DISPLAY:VHHK·VHHK_SCT_07_DEH_03·135·250:VHHK·VHHK_SCT_07_DEH_03·135·250:VHHK·VHHK_SCT_25_DEH_02·100·250
 
+SECTORLINE:4c ;07 FAD South and East (NAP)
+COORD:N022.14.37.575:E113.53.26.203
+COORD:N021.59.13.078:E113.49.05.344
+COORD:N021.57.04.363:E113.43.57.910
+DISPLAY:VHHK·VHHK_SCT_07_FAD_01·000·100:VHHK·VHHK_SCT_07_FAD_01·000·100:VHHK·VHHK_SCT_07NAP_APP_01·000·135
+DISPLAY:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:VHHK_SCT_07NAP_APP_01·000·135
```

Comment:
Are we missing the reverse configuration line here?

Reply:
![image](https://github.com/vatsimhk/Hong-Kong-Sector-Package/assets/89831445/debe445b-e353-4a19-a526-576d74333853)
![image](https://github.com/vatsimhk/Hong-Kong-Sector-Package/assets/89831445/7e1ec30c-8397-444d-8c35-bb1546ec5967)


it should already be covered by the non-nap variant of the line as whoever owns FAD (no matter NAP or DAY) will own VHHK_SCT_07_FAD_01

Reply:
I mean should these display too
```suggestion
DISPLAY:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:VHHK_SCT_07NAP_APP_01·000·135
DISPLAY:VHHK_SCT_07NAP_APP_01·000·135:VHHK·VHHK_SCT_07NAP_DEH_01·100·250:VHHK_SCT_07NAP_APP_01·000·135
```

Reply:
ohh sorry I see it now - these overlap with sectorlines 113 and 114 so they don't appear to do anything
but let's add them in I suppose

Reply:
yeah tbh i tested it in sweatbox and it appears to make no difference at all, but for completeness let's just add it